runners/extensions-java/metrics/src/test/java/org/apache/beam/runners/extensions/metrics/MetricsGraphiteSinkTest.java

```diff
@@ -69,20 +69,20 @@ public void testWriteMetricsWithCommittedSupported() throws Exception {
     countDownLatch.await();
     String join = String.join("\n", graphiteServer.getMessages());
     String regexpr =
-        "beam.counter.ns1.n1.committed.value 10 [0-9]+\\n"
-            + "beam.counter.ns1.n1.attempted.value 20 [0-9]+\\n"
-            + "beam.gauge.ns1.n3.committed.value 100 [0-9]+\\n"
-            + "beam.gauge.ns1.n3.attempted.value 120 [0-9]+\\n"
-            + "beam.distribution.ns1.n2.committed.min 5 [0-9]+\\n"
-            + "beam.distribution.ns1.n2.attempted.min 3 [0-9]+\\n"
-            + "beam.distribution.ns1.n2.committed.max 8 [0-9]+\\n"
-            + "beam.distribution.ns1.n2.attempted.max 9 [0-9]+\\n"
-            + "beam.distribution.ns1.n2.committed.count 2 [0-9]+\\n"
-            + "beam.distribution.ns1.n2.attempted.count 4 [0-9]+\\n"
-            + "beam.distribution.ns1.n2.committed.sum 10 [0-9]+\\n"
-            + "beam.distribution.ns1.n2.attempted.sum 25 [0-9]+\\n"
-            + "beam.distribution.ns1.n2.committed.mean 5.0 [0-9]+\\n"
-            + "beam.distribution.ns1.n2.attempted.mean 6.25 [0-9]+";
+        "beam.counter.ns1.n1.s1.committed.value 10 [0-9]+\\n"
```

Comment:
What is a reason for this and all other similar changes (`n1` -> `s1`)?

Reply:
s1 is the step that declares the counter, s2 the step that declares distribution and s3 is the step that declares gauge.
All the point of this PR is to add missing step names in the graphite sink message format. So we need to update the test as well to expect the step names in the output.